packager/packager.cc

```diff
@@ -603,7 +605,12 @@ unique_ptr<PackageInfo> getPackageInfo(core::MutableContext ctx, ast::ParsedFile
     package.tree = ast::TreeMap::apply(ctx, finder, move(package.tree));
     finder.finalize(ctx);
     if (finder.info) {
-        finder.info->packagePathPrefix = packageFilePath.substr(0, packageFilePath.find_last_of('/') + 1);
+        finder.info->packagePathPrefixes.emplace_back(packageFilePath.substr(0, packageFilePath.find_last_of('/') + 1));
+        for (string prefix : extraPackageFilesDirectoryPrefixes) {
+            string_view shortName = finder.info->name.mangledName.shortName(ctx.state);
+            string additionalDirPath = absl::StrCat(prefix, shortName.substr(0, shortName.find("_Package")), "/");
```

Comment:
Is it worth `ENFORCE`'ing that this `find` call doesn't return `shortName.end()`?

Reply:
Can we pull the computation of `shortName` and the `substr`'ing on it out of the loop manually?  That way we're not doing a string `find` every time through the loop.

Reply:
Ah yes, this can be hoisted, will do.

Reply:
Also on the `shortName.end()` `ENFORCE`, I'm not sure if it's worth enforcing here, as we ourselves in the packager are constructing this `NameRef` earlier in the piece with a `_Package` at the end.

Reply:
> Also on the `shortName.end()` `ENFORCE`, I'm not sure if it's worth enforcing here

That seems totally fair. :+1: